lighthouse-plugin-publisher-ads/utils/resource-classification.js

```diff
@@ -221,7 +281,7 @@ function isAdIframe(iframe) {
  * @return {boolean}
  */
 function isImplTag(url) {
```

Comment:
Can you also update isAdTag, isAdScript, isAdTag in this file? We might not need isGptOrAMPTag, etc. if you update those

Reply:
👍

Reply:
I updated these core functions and am now testing each use of `isGptOrAMPTag` I added.

Reply:
I was able to remove `isGptOrAMPTag` entirely, it is not needed.

Reply:
Also removed the unused isAMPOrGptAdRequest